src/main/java/Storage.java

```diff
@@ -0,0 +1,51 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+import java.io.IOException;
+
+public class Storage {
+    private TaskList taskList;
+
+    public Storage(TaskList taskList) {
+        this.taskList = taskList;
+    }
+
+    public void load() throws FileNotFoundException {
+        String filePath = "Data/Duke.txt";
+        File file = new File(filePath);
+        Scanner sc = new Scanner(file);
+        while (sc.hasNext()) {
+            String str = sc.nextLine();
+            String[] splitStr = str.split("\\s+" + "\\|" + "\\s+");
+            String taskType = splitStr[0];
+            String taskStatus = splitStr[1];
+            if (taskType.equals("T")) {
+                ToDo t = new ToDo(splitStr[2]);
+                this.taskList.addTask(t);
+            } else if (taskType.equals("E")) {
+                LocalDateTime dateTime = LocalDateTime.parse(splitStr[3].trim());
```

Comment:
You should avoid the use of magic numbers (in this case splitStr[3]) to avoid confusion when others read your code :)
Maybe assign 3 to an integer to explain what is in that location in splitStr[].
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
Agreed, the use of an enum would be great